python/ray/util/state/common.py

```diff
@@ -93,6 +93,15 @@ def timestamp(x: float):
         """Converts miliseconds to a datetime object."""
         return str(datetime.datetime.fromtimestamp(x / 1000))
 
+    def timestamp_or_empty(x: int):
```

Comment:
Does it make sense to unify `timestamp_or_empty` and `timestamp` into a single method that can take either float or int, valid values and invalid values?

This way user does not need to decide which exact method to call.

Reply:
fixed